tests/phpunit/tests/image/functions.php

```diff
@@ -343,6 +343,25 @@ private function get_image_editor_engine_classes() {
 		return $classes;
 	}
 
+
+	public function test_wp_crop_image_extension() {
+		add_filter( 'image_editor_output_format', array( $this, 'filter_image_editor_output_format' ) );
+		$file = wp_crop_image(
+			DIR_TESTDATA . '/images/canola.jpg',
+			0,
+			0,
+			100,
+			100,
+			100,
+			100
+		);
+		$this->assertNotWPError( $file );
+		$this->assertFileExists( $file );
+
+		unlink( $file );
+		remove_filter( 'image_editor_output_format', array( $this, 'filter_image_editor_output_format' ) );
```

Comment:
```suggestion
```

Filters are automatically removed at the end of tests so you don't need to worry about it.